archlinuxcn/trojan-go-git/PKGBUILD

```diff
@@ -0,0 +1,39 @@
+pkgname=trojan-go-git
+_pkgname=trojan-go
+pkgver=0.6.0.r0.gb93b95f
+pkgrel=1
+pkgdesc="A Trojan proxy written in Go (git version)."
+arch=(x86_64)
+url="https://github.com/p4gefau1t/trojan-go"
+license=(GPL3)
+depends=(glibc v2ray-geoip v2ray-domain-list-community)
+makedepends=(go)
```

Comment:
Build failed.

```
==> Validating source files with sha512sums...
    trojan-go ... Skipped
==> ERROR: Cannot find the git package needed to handle git sources.
```

Reply:
this is fixed in b12f859e398e9a8a2794b5246138be6f1b50489e.